Dockerfile

```diff
@@ -69,23 +72,6 @@ RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl \
     && chmod u+x kubectl && mv kubectl /usr/bin/kubectl
 
-# Install docker
-RUN export DOCKER_VERSION=$(curl --silent --fail --retry 3 https://download.docker.com/linux/static/stable/x86_64/ | grep -o -e 'docker-[.0-9]*-ce\.tgz' | sort -r | head -n 1) \
-    && DOCKER_URL="https://download.docker.com/linux/static/stable/x86_64/${DOCKER_VERSION}" \
-    && curl --silent --show-error --location --fail --retry 3 --output /tmp/docker.tgz "${DOCKER_URL}" \
-    && tar -xz -C /tmp -f /tmp/docker.tgz \
-    && mv /tmp/docker/* /usr/bin \
-    && rm -rf /tmp/docker /tmp/docker.tgz
-
-# Install docker-compose
-RUN python3 -m ensurepip --upgrade --default-pip && python3 -m pip install --upgrade pip && python3 -m pip install PyYAML -U && pip3 install docker-compose
-
-
-# Install git-crypt
-RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-git-crypt/master/sgerrand.rsa.pub && \
-    wget https://github.com/sgerrand/alpine-pkg-git-crypt/releases/download/0.6.0-r1/git-crypt-0.6.0-r1.apk && \
-    rm git-crypt-0.6.0-r1.apk
-
 # Install AWS CLI
 ENV PATH /root/.local/bin:$PATH
 RUN pip3 install awscli --upgrade --user
```

Comment:
Worth removing this - just thinking.  We deploy nothing to AWS as far as I'm away (from a Repo perspective) - just to make it smaller.

Reply:
Kubectl will be able to go in the coming weeks, too.